int32 id

float32 x
float32 y
float32 width
float32 height

float32 semantic_confidence
string semantic_name
uint32 semantic_id

uint32 r
uint32 g
uint32 b